test_scripts/Polices/External_UCS/003_ATF_P_TC_LPT_Creation_PreloadedPT_without_disallowed_by_external_consent_entities_off_struct.lua

```diff
@@ -0,0 +1,137 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: PreloadedPT without "disallowed_by_external_consent_entities_off" struct
+--
+-- Description:
+-- In case:
+-- SDL uploads PreloadedPolicyTable without "disallowed_by_external_consent_entities_off:
+-- [entityType: <Integer>, entityId: <Integer>]" -> of "<functional grouping>" -> from "functional_groupings" section
+-- SDL must:
+-- a. consider this PreloadedPT as valid (with the pre-conditions of all other valid PreloadedPT content)
+-- b. do not create this "disallowed_by_external_consent_entities_off: [entityType: <Integer>, entityId: <Integer>]"
+-- field of the corresponding "<functional grouping>" in the Policies database.
+--
+-- Preconditions:
+-- 1. Stop SDL
+-- 2. Modify PreloadedPolicyTable (remove 'disallowed_by_external_consent_entities_off' section)
+--
+-- Steps:
+-- 1. Start SDL
```

Comment:
@dboltovskyi, will be good to specify that SDL is built with External_Proprietary flag.